### PR TITLE
support cuda

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@jakirkham, @kkraus14, @jaimergp have you seen the following error before?

> "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\bin\nvcc.exe" "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\include" -Isrc -c src/vkfft_cuda.cu -o build\temp.win-amd64-cpython-310\Release\src/vkfft_cuda.o -O3 --ptxas-options=-v -Xcompiler /MD -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64"
>  'cl.exe' is not recognized as an internal or external command,
> operable program or batch file.

**Comment by kkraus14** [comment]
> @jakirkham, @kkraus14, @jaimergp have you seen the following error before?
> 
> > "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\bin\nvcc.exe" "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\include" -Isrc -c src/vkfft_cuda.cu -o build\temp.win-amd64-cpython-310\Release\src/vkfft_cuda.o -O3 --ptxas-options=-v -Xcompiler /MD -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64"
> > 'cl.exe' is not recognized as an internal or external command,
> > operable program or batch file.

I unfortunately have not, but I've done close to 0 Windows development 😅

**Comment by jakirkham** [comment]
Sorry I haven't either. Would defer to Jaime who has done a fair bit of work with CUDA on Windows.

**Comment by jaimergp** [comment]
Hm, I haven't seen this, but my experience with Windows/CUDA involved CMake primarily.

That said:

```
"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\bin\nvcc.exe" ^
  "-IC:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.2\include" ^
  -Isrc ^
  -c src/vkfft_cuda.cu ^
  -o build\temp.win-amd64-cpython-37\Release\src/vkfft_cuda.o ^
  -O3 ^
  --ptxas-options=-v ^
  -Xcompiler /MD ^
  -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64"
```

... could it be that `-ccbin` is added too late to the command line? Maybe everything after `-Xcompiler` gets treated as host compiler options? What if you move the `ccbin` part to earlier in the command? Our [wrapper](https://github.com/conda-forge/nvcc-feedstock/blob/6aa4413734dc43299bb948d561073e28d1f8d5e7/recipe/windows/nvcc_windows.bat#L1) puts it right after `nvcc.exe`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,22 +10,26 @@ source:
...
-  number: 1
+  number: 2
   entry_points:
     - pyvkfft-test = pyvkfft.scripts.pyvkfft_test:main
   script:
-    - export VKFFT_BACKEND=opencl  # [unix]
-    - set VKFFT_BACKEND=opencl     # [win]
-    - {{ PYTHON }} -m pip install . -vv
+    - set "CC="    # [win]
+    - set "CXX="   # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:21]:
```suggestion
```

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Pull request merged**